From 0e221fcd66697a232c07b5adaf60a57da7bd1005 Mon Sep 17 00:00:00 2001
From: Adeli Yeh <Adeli_Yeh@phoenix.com>
Date: Thu, 4 Nov 2021 10:02:00 +0800
Subject: [PATCH] Shorten the name to keep the length below 16 as much as
 possible to avoid exceeding the IPMI SDR id string limit.

---
 src/PSUSensorMain.cpp | 118 +++++++++++++++++++++---------------------
 1 file changed, 59 insertions(+), 59 deletions(-)
 mode change 100644 => 100755 src/PSUSensorMain.cpp

diff --git a/src/PSUSensorMain.cpp b/src/PSUSensorMain.cpp
old mode 100644
new mode 100755
index e979189..ebf576f
--- a/src/PSUSensorMain.cpp
+++ b/src/PSUSensorMain.cpp
@@ -916,65 +916,65 @@ void propertyInitialize(void)
                    {"fan", sensor_paths::unitRPMs}};
 
     labelMatch = {
-        {"pin", PSUProperty("Input Power", 3000, 0, 6, 0)},
-        {"pout1", PSUProperty("Output Power", 3000, 0, 6, 0)},
-        {"pout2", PSUProperty("Output Power", 3000, 0, 6, 0)},
-        {"pout3", PSUProperty("Output Power", 3000, 0, 6, 0)},
-        {"power1", PSUProperty("Output Power", 3000, 0, 6, 0)},
-        {"maxpin", PSUProperty("Max Input Power", 3000, 0, 6, 0)},
-        {"vin", PSUProperty("Input Voltage", 300, 0, 3, 0)},
-        {"maxvin", PSUProperty("Max Input Voltage", 300, 0, 3, 0)},
-        {"vout1", PSUProperty("Output Voltage", 255, 0, 3, 0)},
-        {"vout2", PSUProperty("Output Voltage", 255, 0, 3, 0)},
-        {"vout3", PSUProperty("Output Voltage", 255, 0, 3, 0)},
-        {"vout4", PSUProperty("Output Voltage", 255, 0, 3, 0)},
-        {"vout5", PSUProperty("Output Voltage", 255, 0, 3, 0)},
-        {"vout6", PSUProperty("Output Voltage", 255, 0, 3, 0)},
-        {"vout7", PSUProperty("Output Voltage", 255, 0, 3, 0)},
-        {"vout8", PSUProperty("Output Voltage", 255, 0, 3, 0)},
-        {"vout9", PSUProperty("Output Voltage", 255, 0, 3, 0)},
-        {"vout10", PSUProperty("Output Voltage", 255, 0, 3, 0)},
-        {"vout11", PSUProperty("Output Voltage", 255, 0, 3, 0)},
-        {"vout12", PSUProperty("Output Voltage", 255, 0, 3, 0)},
-        {"vout13", PSUProperty("Output Voltage", 255, 0, 3, 0)},
-        {"vout14", PSUProperty("Output Voltage", 255, 0, 3, 0)},
-        {"vout15", PSUProperty("Output Voltage", 255, 0, 3, 0)},
-        {"vout16", PSUProperty("Output Voltage", 255, 0, 3, 0)},
-        {"vout17", PSUProperty("Output Voltage", 255, 0, 3, 0)},
-        {"vout18", PSUProperty("Output Voltage", 255, 0, 3, 0)},
-        {"vout19", PSUProperty("Output Voltage", 255, 0, 3, 0)},
-        {"vout20", PSUProperty("Output Voltage", 255, 0, 3, 0)},
-        {"vout21", PSUProperty("Output Voltage", 255, 0, 3, 0)},
-        {"vout22", PSUProperty("Output Voltage", 255, 0, 3, 0)},
-        {"vout23", PSUProperty("Output Voltage", 255, 0, 3, 0)},
-        {"vout24", PSUProperty("Output Voltage", 255, 0, 3, 0)},
-        {"vout25", PSUProperty("Output Voltage", 255, 0, 3, 0)},
-        {"vout26", PSUProperty("Output Voltage", 255, 0, 3, 0)},
-        {"vout27", PSUProperty("Output Voltage", 255, 0, 3, 0)},
-        {"vout28", PSUProperty("Output Voltage", 255, 0, 3, 0)},
-        {"vout29", PSUProperty("Output Voltage", 255, 0, 3, 0)},
-        {"vout30", PSUProperty("Output Voltage", 255, 0, 3, 0)},
-        {"vout31", PSUProperty("Output Voltage", 255, 0, 3, 0)},
-        {"vout32", PSUProperty("Output Voltage", 255, 0, 3, 0)},
-        {"vmon", PSUProperty("Auxiliary Input Voltage", 255, 0, 3, 0)},
-        {"in1", PSUProperty("Output Voltage", 255, 0, 3, 0)},
-        {"iin", PSUProperty("Input Current", 20, 0, 3, 0)},
-        {"iout1", PSUProperty("Output Current", 255, 0, 3, 0)},
-        {"iout2", PSUProperty("Output Current", 255, 0, 3, 0)},
-        {"iout3", PSUProperty("Output Current", 255, 0, 3, 0)},
-        {"iout4", PSUProperty("Output Current", 255, 0, 3, 0)},
-        {"iout5", PSUProperty("Output Current", 255, 0, 3, 0)},
-        {"iout6", PSUProperty("Output Current", 255, 0, 3, 0)},
-        {"iout7", PSUProperty("Output Current", 255, 0, 3, 0)},
-        {"iout8", PSUProperty("Output Current", 255, 0, 3, 0)},
-        {"iout9", PSUProperty("Output Current", 255, 0, 3, 0)},
-        {"iout10", PSUProperty("Output Current", 255, 0, 3, 0)},
-        {"iout11", PSUProperty("Output Current", 255, 0, 3, 0)},
-        {"iout12", PSUProperty("Output Current", 255, 0, 3, 0)},
-        {"iout13", PSUProperty("Output Current", 255, 0, 3, 0)},
-        {"iout14", PSUProperty("Output Current", 255, 0, 3, 0)},
-        {"curr1", PSUProperty("Output Current", 255, 0, 3, 0)},
-        {"maxiout1", PSUProperty("Max Output Current", 255, 0, 3, 0)},
+        {"pin", PSUProperty("In Power", 3000, 0, 6, 0)},
+        {"pout1", PSUProperty("Out Power", 3000, 0, 6, 0)},
+        {"pout2", PSUProperty("Out Power", 3000, 0, 6, 0)},
+        {"pout3", PSUProperty("Out Power", 3000, 0, 6, 0)},
+        {"power1", PSUProperty("Out Power", 3000, 0, 6, 0)},
+        {"maxpin", PSUProperty("Max In Power", 3000, 0, 6, 0)},
+        {"vin", PSUProperty("In Voltage", 300, 0, 3, 0)},
+        {"maxvin", PSUProperty("Max In Voltage", 300, 0, 3, 0)},
+        {"vout1", PSUProperty("Out Voltage", 255, 0, 3, 0)},
+        {"vout2", PSUProperty("Out Voltage", 255, 0, 3, 0)},
+        {"vout3", PSUProperty("Out Voltage", 255, 0, 3, 0)},
+        {"vout4", PSUProperty("Out Voltage", 255, 0, 3, 0)},
+        {"vout5", PSUProperty("Out Voltage", 255, 0, 3, 0)},
+        {"vout6", PSUProperty("Out Voltage", 255, 0, 3, 0)},
+        {"vout7", PSUProperty("Out Voltage", 255, 0, 3, 0)},
+        {"vout8", PSUProperty("Out Voltage", 255, 0, 3, 0)},
+        {"vout9", PSUProperty("Out Voltage", 255, 0, 3, 0)},
+        {"vout10", PSUProperty("Out Voltage", 255, 0, 3, 0)},
+        {"vout11", PSUProperty("Out Voltage", 255, 0, 3, 0)},
+        {"vout12", PSUProperty("Out Voltage", 255, 0, 3, 0)},
+        {"vout13", PSUProperty("Out Voltage", 255, 0, 3, 0)},
+        {"vout14", PSUProperty("Out Voltage", 255, 0, 3, 0)},
+        {"vout15", PSUProperty("Out Voltage", 255, 0, 3, 0)},
+        {"vout16", PSUProperty("Out Voltage", 255, 0, 3, 0)},
+        {"vout17", PSUProperty("Out Voltage", 255, 0, 3, 0)},
+        {"vout18", PSUProperty("Out Voltage", 255, 0, 3, 0)},
+        {"vout19", PSUProperty("Out Voltage", 255, 0, 3, 0)},
+        {"vout20", PSUProperty("Out Voltage", 255, 0, 3, 0)},
+        {"vout21", PSUProperty("Out Voltage", 255, 0, 3, 0)},
+        {"vout22", PSUProperty("Out Voltage", 255, 0, 3, 0)},
+        {"vout23", PSUProperty("Out Voltage", 255, 0, 3, 0)},
+        {"vout24", PSUProperty("Out Voltage", 255, 0, 3, 0)},
+        {"vout25", PSUProperty("Out Voltage", 255, 0, 3, 0)},
+        {"vout26", PSUProperty("Out Voltage", 255, 0, 3, 0)},
+        {"vout27", PSUProperty("Out Voltage", 255, 0, 3, 0)},
+        {"vout28", PSUProperty("Out Voltage", 255, 0, 3, 0)},
+        {"vout29", PSUProperty("Out Voltage", 255, 0, 3, 0)},
+        {"vout30", PSUProperty("Out Voltage", 255, 0, 3, 0)},
+        {"vout31", PSUProperty("Out Voltage", 255, 0, 3, 0)},
+        {"vout32", PSUProperty("Out Voltage", 255, 0, 3, 0)},
+        {"vmon", PSUProperty("Aux In Voltage", 255, 0, 3, 0)},
+        {"in1", PSUProperty("Out Voltage", 255, 0, 3, 0)},
+        {"iin", PSUProperty("In Current", 20, 0, 3, 0)},
+        {"iout1", PSUProperty("Out Current", 255, 0, 3, 0)},
+        {"iout2", PSUProperty("Out Current", 255, 0, 3, 0)},
+        {"iout3", PSUProperty("Out Current", 255, 0, 3, 0)},
+        {"iout4", PSUProperty("Out Current", 255, 0, 3, 0)},
+        {"iout5", PSUProperty("Out Current", 255, 0, 3, 0)},
+        {"iout6", PSUProperty("Out Current", 255, 0, 3, 0)},
+        {"iout7", PSUProperty("Out Current", 255, 0, 3, 0)},
+        {"iout8", PSUProperty("Out Current", 255, 0, 3, 0)},
+        {"iout9", PSUProperty("Out Current", 255, 0, 3, 0)},
+        {"iout10", PSUProperty("Out Current", 255, 0, 3, 0)},
+        {"iout11", PSUProperty("Out Current", 255, 0, 3, 0)},
+        {"iout12", PSUProperty("Out Current", 255, 0, 3, 0)},
+        {"iout13", PSUProperty("Out Current", 255, 0, 3, 0)},
+        {"iout14", PSUProperty("Out Current", 255, 0, 3, 0)},
+        {"curr1", PSUProperty("Out Current", 255, 0, 3, 0)},
+        {"maxiout1", PSUProperty("Max Out Current", 255, 0, 3, 0)},
         {"temp1", PSUProperty("Temperature", 127, -128, 3, 0)},
         {"temp2", PSUProperty("Temperature", 127, -128, 3, 0)},
         {"temp3", PSUProperty("Temperature", 127, -128, 3, 0)},
